mrp_production_batch/models/stock_picking.py

```diff
@@ -0,0 +1,78 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import fields, models
+
+
+class PickingType(models.Model):
+    _inherit = "stock.picking.type"
```

Comment:
This model is not `stock.picking` it should be in a separate file `stock_picking_type`. We use OCA guidelines not the Odoo ones...